schema.sql

```diff
@@ -1,11 +1,34 @@
-/* Database schema to keep the structure of entire database. */
+-- These scripts will delete the tables if it's already there
+DROP TABLE IF EXISTS owners;
+DROP TABLE IF EXISTS species;
+DROP TABLE IF EXISTS animals;
 
-CREATE TABLE animals (
-    id INTEGER PRIMARY KEY,
+-- Create owners table
+CREATE TABLE owners
+(
+	owner_id INTEGER PRIMARY KEY,
```

Comment:
- [ ] When you make a column like this for `owner_id`, you will have to manually set the value in your _**data.sql**_ file. I would advise you to make it automatic. You could do this `owner_id SERIAL PRIMARY KEY`. The reason I asked you for this change is that when I run the **_data.sql file_**, I get this error:

![image](https://user-images.githubusercontent.com/77106962/165581581-72239538-81de-4267-9a5b-6ae3d8ae2fa9.png)